.github/workflows/ci.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
       - uses: golangci/golangci-lint-action@v2
         with:
           only-new-issues: true
-          args: --timeout=5m
+          args: --timeout=5m --out-${NO_FUTURE}format colored-line-number
```

Comment:
I would prefer to avoid using a flag where the tooling's guidance is to avoid using it, because it's not guaranteed to be stable going forward. How about running `make lint` instead?